docs/docsite/rst/porting_guides/porting_guide_5.rst

```diff
@@ -492,7 +492,7 @@ amazon.aws
 arista.eos
 ~~~~~~~~~~
 
-- Arista released train 4.23.X and newer and along with it replaced and deprecated lots of commands. This PR adds support for syntax changes in release train 4.23 and after. Going forward the eos modules will not support eos sw version < 4.23.
+- Arista released train 4.23.X and newer and along with it replaced and deprecated several commands. This PR adds support for syntax changes in release train 4.23 and after. Going forward the eos modules will not support eos sw version < 4.23.
```

Comment:
Remove the porting guide files. They are autogenerated.

Reply:
Normally you are correct @Akasurde but we are no longer generating old porting guides like this so will let this fix stay.